signing-and-verification-workflow.md

```diff
@@ -0,0 +1,52 @@
+# Signing and Verification Workflow
+
+This document describes how Notary v2 signs and verifies OCI artifacts.
+
+## Signing workflow
+
+The user wants to sign a locally available OCI artifact and push it to a repository(along with signature).
+
+### Prerequisite
+
+- User has access to the signing certificate and private key.
+
+### Steps
+
+1. **Generate signature:** Using notation CLI or any other compliant signing tool, sign the OCI artifact. The signing tool should follow the following guideline.
+    1. Verify that the signing certificate is valid and satisfies [certificate requirements](./signature-specification.md#certificate-requirements).
```

Comment:
While I agree in concept, is this a requirement of the notation client? Is the notation client expected to "verify..."? Or, are we saying

> The signing certificate MUST satisfy the [certificate requirements](./signature-specification.md#certificate-requirements).

Reply:
This document depicts all the steps required for signature generation and verification. Also, the compliant client MUST perform these steps. 

I am not sure how to combine workflow and specification/requirement so wrote the complete doc as workflow/control-flow document. Also, its easier to read and follow workflow than specification.